sema4ai/vscode-client/src/viewsRobots.ts

```diff
@@ -275,6 +275,15 @@ export class RobotsTreeDataProvider implements vscode.TreeDataProvider<RobotEntr
                         "parent": element,
                         "tooltip": "Publishes the Action Package to the Sema4.ai Desktop application",
                     },
+                    {
+                        "label": "Build Package to Workspace",
+                        "uri": element.uri,
+                        "robot": element.robot,
+                        "iconPath": "archive",
+                        "type": RobotEntryType.PackageBuildToWorkspace,
+                        "parent": element,
+                        "tooltip": "Build the Action Package .zip file to the workspace folder",
```

Comment:
The tooltip needs to follow the `label` copy to some extend.
Might just be that the only thing that needs to change would he the first part of the text.